.travis.yml

```diff
@@ -20,8 +20,9 @@ install:
 
 script:
  - cabal-3.0 configure --enable-tests
- - cabal-3.0 build
- - cabal-3.0 test --test-show-details=streaming
- - cabal-3.0 check
- - cabal-3.0 haddock
- - cabal-3.0 sdist
+ - pushd bech32    && cabal-3.0 check && popd
+ - pushd bech32-th && cabal-3.0 check && popd
```

Comment:
Unfortunately we can't run a `cabal check all` here, as this is not yet supported. (See https://github.com/haskell/cabal/issues/6282)

Reply:
`check` may not work, but what about `test` ?

eg https://github.com/HuwCampbell/grenade/blob/master/.travis.yml#L27-L29

Reply:
> `check` may not work, but what about `test` ?

The `test` command still works. See a couple of lines down:

```
  - cabal-3.0 test all --test-show-details=streaming
```

Reply:
Ah, that's right, `check` just checks that the packaging info is right. Never mind me. Yes I think your solution works, but so does:
```
(cd bech32  && cabal-3.0 check)
(cd bech32-th && cabal-3.0 check)
```
and has better behavior in the failure case.

Reply:
> (cd bech32  && cabal-3.0 check)
> (cd bech32-th && cabal-3.0 check)

That's a great suggestion. I'll make the change: :+1:

Reply:
@erikd: Fixed in https://github.com/input-output-hk/bech32/pull/15/commits/1f78b8e588383cfbec0eced7ac99141940c2e494.